File: Problem2/d2.java
import java.util.Scanner;

public class d2 {
    public static void main(String[] args) {
        Scanner input = new Scanner(System.in);

        int n = input.nextInt();           
        long totalMilk = 0;              

        int[] milk = new int[n];           
        for (int i = 0; i < n; i++) {
            milk[i] = input.nextInt();
            totalMilk += milk[i];         
        }

     
        if (totalMilk % n == 0) {
            System.out.println("YES");
        } else {
            System.out.println("NO");
        }
        input.close(); 
    }

}

File: Problem2/d2.cpp
#include <iostream>
using namespace std;

int main() {
    int n;
    cin >> n;

    long long totalMilk = 0;   
    int milkInGlass;

    
    for (int i = 0; i < n; i++) {
        cin >> milkInGlass;
        totalMilk += milkInGlass;
    }

    if (totalMilk % n == 0) {
        cout << "YES" << endl;
    } else {
        cout << "NO" << endl;
    }

    return 0;
}